__tests__/app.test.js

```diff
@@ -109,3 +108,51 @@ describe('GET /api', () => {
         });
     })
 })
+describe('GET /api/articles', () => {
+  describe('Basic request checks', () => {
+    test('returns status 200 on successful request', () => {
+      return request(app)
+      .get('/api/articles')
+      .expect(200);
+    });
+    test('responds with an array of all article objects with expected properties', () => {
+      return request(app)
+        .get('/api/articles')
+        .then(({ body }) => {
+          const expectedProperties = [
+            'author',
+            'title',
+            'article_id',
+            'topic',
+            'created_at',
+            'votes',
+            'article_img_url',
+            'comment_count',
+          ];
+          body.articles.forEach(article => {
+            expectedProperties.forEach(property => {
+              expect(article).toHaveProperty(property);
```

Comment:
We should also check that the datatypes of each property are what we expect them to be. I would suggest using a flexible jest matcher such as toMatchObject here instead which would allow you to check both the property and the expected data type at once.